src/fem/dh_cell_accessor.hh

```diff
@@ -190,6 +192,36 @@ public:
     	return cell().dim();
     }
 
+    /// Side centre.
+    inline arma::vec3 centre() const {
+    	return side().centre();
+    }
+
+    inline ElementAccessor<3> element() const {
+    	return side().element();
+    }
+
+    inline unsigned int elem_idx() const {
+    	return side().elem_idx();
+    }
+
+    inline Boundary * cond() const {
```

Comment:
Is this the problematic implementation in the release build? If it is the case, we should rather use the temporary fixed implementation until the issue #1114  is resolved.

Reply:
Solved by cherry-pick of appropriate commit.